selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/gwt/CheckSimpleGwtAppTest.java

```diff
@@ -99,7 +99,9 @@ public void checkLaunchingCodeServer() throws Exception {
     consoles.waitExpectedTextIntoConsole("The code server is ready", APPLICATION_START_TIMEOUT_SEC);
 
     String url =
-        workspaceServiceClient.getServerByExposedPort(testWorkspace.getId(), "9876/tcp").getUrl();
+        workspaceServiceClient
+            .getServerFromDevMachineBySymbolicName(testWorkspace.getId(), "9876/tcp")
```

Comment:
Are you sure that this symbolic name won't change? AFAIK we don't have a policy on how we create servers for exposed ports if the user has not provided his own server.
So, I suggest adding the server into workspace config and use it instead.

Reply:
ok